<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeScreen</class>
 <widget class="QDialog" name="HomeScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>1130</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>1130</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>720</width>
     <height>1280</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>4</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="maximumSize">
     <size>
      <width>720</width>
      <height>1280</height>
     </size>
    </property>
    <widget class="QPushButton" name="buttonViewTestResults">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>480</y>
       <width>325</width>
       <height>325</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 70%;
background-color: rgb(255, 255, 255,0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/Images/XatekUI_ViewTestResults.png</normaloff>:/Images/XatekUI_ViewTestResults.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>325</width>
       <height>325</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonStartNewTest">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>325</width>
       <height>325</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 70%;
background-color: rgb(255, 255, 255,0);
Layer.enabled: rounded;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/Images/XatekUI_StartNewTest.png</normaloff>:/Images/XatekUI_StartNewTest.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>325</width>
       <height>325</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="labelNewTest">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>360</y>
       <width>720</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>40</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255,0);
color: white;</string>
     </property>
     <property name="text">
      <string>New Test</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelTestResults">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>820</y>
       <width>720</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>44</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255,0);
color: white;</string>
     </property>
     <property name="text">
      <string>Test Results</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>buttonStartNewTest</tabstop>
  <tabstop>buttonViewTestResults</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
